### PR TITLE
Update security risks statement with link based on feedback

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

**Comment by mattwojo** [comment]
@v-britweston made updates based on discussion with Patricia Ricciardi

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-antivirus-performance-mode.md [hunk 1]
```diff
@@ -79,20 +79,7 @@ For requirements specific to Dev Drive, see [Set up a Dev Drive on Windows 11](/
...
-
-A Dev Drive is automatically designated as trusted, providing the best possible performance by default. A trusted Dev Drive means that the developer using the volume has high confidence in the security of the content stored there.
-
-Similar to when an administrator chooses to add an exclusion to a Microsoft Defender Antivirus configuration, it's the administrator's responsibility to assess the performance benefits and security risks when using performance mode. As mentioned, security runs in Real-time protection mode when a Dev Drive is untrusted. 
-
-> [!NOTE]
-> Performance mode can only run on a “trusted” Dev Drive.
-
-For more information on trusted Dev Drive and to confirm whether a designated Dev Drive is trusted, see: [Set up a Dev Drive on Windows 11](/windows/dev-drive).
+Performance mode can only run on a “trusted” Dev Drive and is enabled by default when a new Dev Drive is created. For more information, see: [Understanding security risks and trust in relation to Dev Drive](/windows/dev-drive#understanding-security-risks-and-trust-in-relation-to-dev-drive).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-antivirus-performance-mode.md]:
```suggestion
Performance mode can only run on a *trusted* Dev Drive and is enabled by default when a new Dev Drive is created. For more information, see [Understanding security risks and trust in relation to Dev Drive](/windows/dev-drive#understanding-security-risks-and-trust-in-relation-to-dev-drive).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mattwojo, kindly commit the suggested minor changes. Many thanks.

**Review by v-britweston — APPROVED** [review]
looks good

**Pull request merged**